src/cnd/cnd.ts

```diff
@@ -277,23 +289,53 @@ export class Cnd {
     return info.listen_addresses;
   }
 
+  /**
+   * Initiate a swap request on cnd.
```

Comment:
```suggestion
   * Initiate a swap on cnd.
```

Initiate and request have a similar meaning, we should use either one or the other.

Initiating a request is something different :)